tests/baselines/reference/fixingTypeParametersRepeatedly2.errors.txt

```diff
@@ -24,6 +28,11 @@ fixingTypeParametersRepeatedly2.ts(17,5): error TS2403: Subsequent variable decl
     declare function bar<T>(x: T, func: (p: T) => T): T;
     declare function bar<T>(x: T, func: (p: T) => T): T;
     var result = bar(derived, d => d.toBase());
-        ~~~~~~
-!!! error TS2403: Subsequent variable declarations must have the same type.  Variable 'result' must be of type 'Derived', but here has type 'Base'.
-!!! related TS6203 fixingTypeParametersRepeatedly2.ts:11:5: 'result' was also declared here.
\ No newline at end of file
+                                   ~~~~~~~~~~

```

Comment:
It's funny - this test seems to insist the overload case here and the no overload case above should have had similar errors - but that actually wasn't true until this PR. Maybe it was a forward looking comment? 🤷 This is certainly more consistent now - both overloads produce the same `Base` is not assignable to `Derived` error.